app/assets/javascripts/admin/addon/mixins/penalty-controller.js

```diff
@@ -66,6 +66,8 @@ export default Mixin.create(ModalFunctionality, {
           callback(result);
         }
       })
-      .catch(popupAjaxError);
+      .catch((error) => {
+        this.set("errorMessage", extractError(error));
```

Comment:
This is minor, but we should initialize `errorMessage: null` in the mixin. It's a good habit to avoid changing the shape of the javascript objects later.

Reply:
I missed that. Thank you!